{
  "file": "core/src/convert/num.rs",
  "name": "convert::num::ptr_try_from_impls::<impl convert::TryFrom<u16> for isize>::try_from",
  "hash": "1070469087219879266417019146462179115889",
  "hash_direct": "11452689472214189475741260095451608580",
  "src": "fn try_from(value: $source) -> Result<Self, Self::Error> {\n                Ok(value as Self)\n            }",
  "macro_backtrace": [
    {
      "callsite": "$mac!($target => $source)",
      "defsite": "macro_rules! impl_try_from_unbounded {\n    ($source:ty => $($target:ty),+) => {$(\n        #[stable(feature = \"try_from\", since = \"1.34.0\")]\n        #[rustc_const_unstable(feature = \"const_try\", issue = \"74935\")]\n        impl const TryFrom<$source> for $target {\n            type Error = TryFromIntError;\n\n            /// Tries to create the target number type from a source\n            /// number type. This returns an error if the source value\n            /// is outside of the range of the target type.\n            #[inline]\n            fn try_from(value: $source) -> Result<Self, Self::Error> {\n                Ok(value as Self)\n            }\n        }\n    )*}\n}"
    },
    {
      "callsite": "rev!(impl_try_from_unbounded, isize => u16, u32)",
      "defsite": "macro_rules! rev {\n    ($mac:ident, $source:ty => $($target:ty),+) => {$(\n        $mac!($target => $source);\n    )*}\n}"
    }
  ]
}
